go/enclave/enclave_admin_service.go

```diff
@@ -300,7 +300,8 @@ func (e *enclaveAdminService) CreateRollup(ctx context.Context, fromSeqNo uint64
 		return nil, responses.ToInternalError(fmt.Errorf("not initialised yet"))
 	}
 
-	result, err := e.sequencer().CreateRollup(ctx, fromSeqNo)
+	// we use the sequencer service directly because backup sequencers don't have the service active, but they can still create rollups
+	result, err := e.sequencerService.CreateRollup(ctx, fromSeqNo)
```

Comment:
The backup sequencer normally uses a 'validator' service for all its processing because it mostly behaves like a validator but the rollup creation is on the 'sequencer' service.

At this point in this method we have already checked the enclave is permissioned as a sequencer, accessing the CreateRollup function here on the sequencerService directly avoids us rethinking the nodetype interfaces.

Reply:
looks reasonable